rust/capture/src/limiters/redis.rs

```diff
@@ -185,15 +147,12 @@ mod tests {
             .zrangebyscore_ret("@posthog/quota-limits/events", vec![String::from("banana")]);
         let client = Arc::new(client);
 
-        let limiter = RedisLimiter::new(Duration::microseconds(1), client, None)
+        let limiter = RedisLimiter::new(Duration::seconds(1), client, None, QuotaResource::Events)
```

Comment:
duration here doesn't really matter because we update immediately. At 1 microsecond we were updating hundreds of times for no reason (we don't only update on fetch now)